### ISSUE TITLE
Repeated line in Inference.jl: @eval getspace(::Type{<:$alg{space}}) where {space} = space

### ISSUE THREAD

**Issue by truedichotomy** [TuringLang/Turing.jl#887]
I'm new to Turing, just tried to install and compile it in Julia 1.2 RC3 and got the following warning: 
```
WARNING: Method definition getspace(Type{#s362} where #s362<:(Turing.Inference.SMC{space, F} where F)) where {space} in module Inference at /Users/gong/.julia/packages/Turing/eIa7P/src/inference/Inference.jl:92 overwritten at /Users/gong/.julia/packages/Turing/eIa7P/src/inference/Inference.jl:93.
  ** incremental compilation may be fatally broken for this module **
```
I looked at Inference.jl and saw that line 93 is an exact repeat of line 92:
``` 
@eval getspace(::Type{<:$alg{space}}) where {space} = space
```
I want to make sure that repeated line is not by mistake which may have caused the warning issued. Thanks.

**Comment by cpfiffer** [comment]
Good catch, thanks! I believe this will be addressed when #793 goes through.

**Comment by yebai** [comment]
Is this fixed now?

**Comment by cpfiffer** [comment]
I believe so, this was fixed with #793.

**Comment by cpfiffer** [comment]
Please reopen this if you notice the issue come up again.
